doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "MQTT Client"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a Pub/Sub Integration using the built in MQTT client.
+
+<!--more-->
+
+Creating a Pub/Sub Integration with the built in MQTT client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client works with any MQTT server.
+
+>No API key is needed for Pub/Sub messaging. All messages received by the Application Server are assumed to be authorized, so be sure to configure appropriate security on your MQTT server.
+
+In your application select the **Pub-subs** submenu from the **Integrations** side menu. Clicking on the **+ Add pub-sub** button will open the Pub/Sub creation screen.
+
+Give your Pub/Sub an **ID**, choose a **Message Format**, and enter a **Base Topic** to subscribe or publish to.
+
+Select **MQTT** as provider.
+
+{{< figure src="pubsub-id.png" alt="Pub/Sub creation screen" >}}
+
+If using Client Certificates, enable **Secure**. You will be asked to upload the **Server Root CA Certificate**, the **Client Certificate**, and the **Client Private Key.**
+
+Enter the endpoint of the MQTT server in the **Server URL** field, including the protocol and port. 
+
+Enter the **Client ID** the Application Server should use to authenticate.
+
+Enable **Use Credentials** if your MQTT server uses password authentication. Enter the username and password the Application Server should authenticate with.
+
+{{< figure src="mqtt-config.png" alt="MQTT configuration" >}}
+
+## Message Types
+
+The Application Server can be configured to publish and subscribe to topics for individual events. See [Message Types]({{< relref "../message-types" >}}) for more information about events and topics.
+
+After enabling events, click the **Add pub-sub** button to enable the integration.
+
+## Subscribing to Upstream Traffic
+
+The Application Server publishes messages for any enabled events. For example, when a device sends an uplink, an `uplink` message is published. To view these using a `mosquitto_sub` client connected to your MQTT server:
+
+```bash
+$ mosquitto_sub -t '#' -v
+# base-topic/uplink {"end_device_ids":{"device_id":"dev1","application_ids":{"application_id":"app1"}},"received_at":"2020-05-12T12:23:07.087614Z","uplink_message":{"session_key_id":"AXIDznz4bnQqtW8T3NsIVg==","f_port":1,"f_cnt":327,"frm_payload":"AQ=="}}
+```
+
+## Scheduling Downlinks
+
+You can schedule downlink messages by publishing to a topic. For example, using a `mosquitto_pub` client connected to your MQTT server:
+
+```bash
+$ mosquitto_pub -t 'base-topic/push-subtopic' -m '{"end_device_ids":{"device_id":"dev1","application_ids":{"application_id":"app1"}},"downlinks":[{"f_port":1,"frm_payload":"AA==","priority":"NORMAL"}]}'
```

Comment:
Idem. People may have an external MQTT server running, not necessarily on localhost